README.md

```diff
@@ -43,9 +43,22 @@ The SDO owner services are packaged as a single docker container that can be run
   chmod +x run-sdo-owner-services.sh
   ```
 
-2. Run `./run-sdo-owner-services.sh -h` to see the usage, and set all of the necessary environment variables.
+2. Run `./run-sdo-owner-services.sh -h` to see the usage, and set all of the necessary environment variables. Export your environment variables `HZN_EXCHANGE_URL`, `HZN_FSS_CSSURL`, `HZN_ORG_ID`, and `HZN_EXCHANGE_USER_AUTH` for the subsequent steps.
 
-3. Start the SDO owner services docker container and view the log:
+   ```bash
+   export HZN_EXCHANGE_URL=https://<cluster-url>/edge-exchange/v1
+   export HZN_FSS_CSSURL=https://<cluster-url>/edge-css
+   export HZN_ORG_ID=mycluster
+   export HZN_EXCHANGE_USER_AUTH=iamapikey:<api-key>
+   ```
+
+3. You will also need the `agent-install.crt` file that can be acquired by going through documentation detailing how to [install the Horizon agent on your edge device](https://www-03preprod.ibm.com/support/knowledgecenter/SSFKVV_4.0/devices/installing/registration.html)
```

Comment:
In this tricky balance between open source and proprietary code, which try to avoid having our open source readmes point to the Knowledge Center, which is documentation for our proprietary product. Instead, i suggest:

3. As part of installing the Horizon management hub, you should have run [edgeNodeFiles.sh](https://github.com/open-horizon/anax/blob/master/agent-install/edgeNodeFiles.sh) which created a tar file containing `agent-install.crt`. Use that to export this environment variable: